client/internal/peer/conn.go

```diff
@@ -487,6 +487,8 @@ func (conn *Conn) configureConnection(remoteConn net.Conn, remoteWgPort int, rem
 		return nil, err
 	}
 
+	runtime.GC()
```

Comment:
Is it necessary for all operation systems?

Reply:
Not necessary but having it running after each established connection might be good for memory in other OS as well